app/src/main/java/in/krharsh17/barview_sample/MainActivity.java

```diff
@@ -29,7 +29,7 @@ protected void onCreate(Bundle savedInstanceState) {
         barModels.add(new BarModel(
                 "Samsung",
                 "30.91",
-                getRandomColor(),
+                getRandomColor(new StringBuilder("#ff0000")),
```

Comment:
I'd prefer passing in a `Color` object here.

Reply:
Okay will make changes

Reply:
But if we would pass in a Color object it would be unecessary packing and unpacking of hex values???
Should I pass in a Color object anyway?

Reply:
It would be packing and unpacking, but conventionally, devs are used to passing `Color` objects when colour is required to be passed, so yes, you should pass in a `Color` object

Reply:
Hmm I would do that

Reply:
there is a problem with this approach @krharsh17 since in the library we have been using 24 bit hex literals but the color object will be of 32 bits including the alpha value which even if we dont consider would be there as opaque by default. so what should I do ??

Reply:
Then you should modify the code to use 32 bits colour.

Reply:
But @krharsh17 we are not using the alpha component of the color anywhere in our library why should we refactor every instance of hex literals in the library ??

Reply:
@krharsh should I go with 24 or 32 bits please tell I'll refactor accordingly !!

Reply:
@krharsh  should I go with 24 or 32 bits please tell I'll refactor accordingly !!

Reply:
32 bits please. Lets maintain uniformity

Reply:
Okay @krharsh I will make changes tonight :)